src/sgame/sg_admin.cpp

```diff
@@ -5999,6 +6000,32 @@ bool G_admin_bot( gentity_t *ent )
 			return false;
 		}
 	}
+	else if ( !Q_stricmp( arg1, "skill" ) )
+	{
+		if ( args.Argc() < 3 )
+		{
+			ADMP( QQ( N_( "missing skill." ) ) );
+			BotUsage( ent );
+			return false;
+		}
+		const std::string& skillStr = args.Argv( 2 );
+		int skill = BotSkillFromString( ent, skillStr.c_str() );
+		team_t team = TEAM_NONE;
+		if ( args.Argc() > 3 )
+		{
+			const std::string& teamStr = args.Argv( 3 );
+			team = BG_PlayableTeamFromString( teamStr.c_str() );
+		}
+		for ( int i = 0; i < MAX_CLIENTS; ++i )
+		{
+			if ( team != TEAM_NONE && G_Team( &g_entities[ i ] ) != team )
```

Comment:
I think this is missing an `ent->inuse` check. Consider using ForEntities<ClientComponent>

Reply:
or `for ( gentity_t *ent : iterate_client_entities )` :eyes:

Reply:
Done.